azure-pipelines.yml

```diff
@@ -11,18 +11,17 @@ variables:
   PYTEST_WORKERS: auto
 
 jobs:
-# Mac and Linux use the same template
-- template: ci/azure/posix.yml
+- template: ci/azure/macos.yml
   parameters:
     name: macOS
-    vmImage: macOS-10.14
+    vmImage: macOS-10.15
```

Comment:
@fangchenli can any of this be split off into an easy preliminary?

Reply:
#42419